templates/test/overview_result_table.html.ep

```diff
@@ -15,7 +15,8 @@
             <tr>
                 <td class="name">
                     % my $test_label = text_with_title($config);
-                    % if (my $description = $type_results->{$config}{description}) {
+                    %#% if (my $description = $type_results->{$config =~ s/ \(.*//r}{description}) {
```

Comment:
Who are you and what did you do to @okurz? Disabled code? in a PR? :P

Reply:
the WIP-commit should not have been there. Fixed that, tidied, updated